README.md

```diff
@@ -267,64 +268,147 @@ standard JS template literal syntax:
   the class. It is compiled away, and so it cannot be directly referenced or
   dynamically modified/assigned.
 
-
 ## Reference: built-in helpers, modifiers, components
 
 As implemented as part of the [Strict Mode Templates RFC][rfc-496], the built in
 helpers, modifiers and components are available for import:
 
-* `array` (`import { array } from '@ember/helper';`)
-* `concat` (`import { concat } from '@ember/helper';`)
-* `fn` (`import { fn } from '@ember/helper';`)
-* `get` (`import { get } from '@ember/helper';`)
-* `hash` (`import { hash } from '@ember/helper';`)
-* `on` (`import { on } from '@ember/modifier';`)
-* `Input` (`import { Input } from '@ember/component';`)
-* `LinkTo` (`import { LinkTo } from '@ember/routing';`)
-* `TextArea` (`import { TextArea } from '@ember/component';`)
+- `array` (`import { array } from '@ember/helper';`)
+- `concat` (`import { concat } from '@ember/helper';`)
+- `fn` (`import { fn } from '@ember/helper';`)
+- `get` (`import { get } from '@ember/helper';`)
+- `hash` (`import { hash } from '@ember/helper';`)
+- `on` (`import { on } from '@ember/modifier';`)
+- `Input` (`import { Input } from '@ember/component';`)
+- `LinkTo` (`import { LinkTo } from '@ember/routing';`)
+- `TextArea` (`import { TextArea } from '@ember/component';`)
 
 [rfc-496]: https://github.com/emberjs/rfcs/pull/496
 
-Compatibility
-------------------------------------------------------------------------------
+## Compatibility
 
-* Ember.js v3.27 or above
-* Ember CLI v2.13 or above
-* `ember-cli-htmlbars` 6.0 or above
-* Node.js v12 or above
+- Ember.js v3.27 or above
+- Ember CLI v2.13 or above
+- `ember-cli-htmlbars` 6.0 or above
+- Node.js v12 or above
 
-
-Installation
-------------------------------------------------------------------------------
+## Installation
 
 ```
 ember install ember-template-imports
 ```
 
-Editor Integrations
-------------------------------------------------------------------------------
+## Usage for V2 Addons
+
+V2 addons are build by default with rollup, see
+[blueprint](https://github.com/embroider-build/addon-blueprint). This addon
+provides integrations to rollup to understand and transpile `.gjs` and `.gts`
+files
+
+### Step 1: Install babel plugin
+
+Add the `ember-template-imports` babel plugin to your babel config:
+
+```diff
+'use strict';
+
+module.exports = {
+  presets: ['@babel/preset-typescript'],
+  plugins: [
++    'ember-template-imports/babel-plugin',
```

Comment:
atm, this needs to be:
```
    "ember-template-imports/src/babel-plugin",
```
because the package.json doesn't use an exports map

Reply:
I wish this to be available under the mentioned import path ;)